aqt/helper.py

```diff
@@ -20,20 +20,145 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 import configparser
+import dataclasses
```

Comment:
Ok; I can get rid of this. This was a convenience, not absolutely necessary.

I had expected that since this was passing the CI tests that dataclasses were OK.

Reply:
Current CI run only on 3.8 and 3.9, not 3.6  to reduce load for CI platform and mirror sites. sorry for inconvenience.